charts/histogram/src/chart-settings/label.js

```diff
@@ -1,14 +1,16 @@
-import chartStyleUtils from '@qlik/common/extra/chart-style-utils';
 import { getValueLabelStyle } from '@qlik/common/extra/chart-style-component';
+import Color from '@qlik/common/extra/color';
 
 //
 // Implementation details
 //
 
-function createLabelSettings(layout, theme, chartId, flags) {
+function createLabelSettings(layout, theme, chartId, flags, themeService) {
   const valueLabelSettings = getValueLabelStyle(chartId, theme, layout, flags);
-  const boxFillColor = valueLabelSettings.fill || theme.getColorPickerColor(layout.color.bar.paletteColor);
-
+  const styles = themeService.getStyles();
+  const boxFillColor = valueLabelSettings.fill || styles.label.value.color;
+  const getContrastColor = () =>
+    Color.isDark(layout.color.bar.paletteColor.color) ? styles.label.value.lightColor : styles.label.value.darkColor;
```

Comment:
Can the bar colors can be set through the theme, like bar-chart? If so it would mean `isDark` can get the wrong input color.